### ISSUE TITLE
Otitis

### ISSUE THREAD

**Issue by cmrn-rhi** [obophenotype/human-phenotype-ontology#7114]
**Preferred term label:**
Otitis

**Synonyms**
ear inflammation
ear infection

**Definition (free text, please give PubMed ID)**
Inflammation of infection of the ear.

**Parent term (use hpo.jax.org/app)**
HP:0031703 - Abnormal ear morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
COVID-19

**Your nano-attribution (ORCID)**
0000-0002-9578-0788

**Database cross reference**
[SNOMEDCT_US:43275000](https://phinvads.cdc.gov/vads/ViewCodeSystemConcept.action?oid=2.16.840.1.113883.6.96&code=43275000)

**Comment by pnrobinson** [comment]
This is probably either otitis media or otitis externa. HPO terms exist for both of these. The SNOMED term you cite is refering to otitis externa or variants thereof, so I would suggest coding with https://hpo.jax.org/app/browse/term/HP:0410017

**Comment by cmrn-rhi** [comment]
The SNOWMED term I included is a parent class to a variety of "otitis externa" but also includes sibling classes such as "Otitis media (disorder)". Our intention was to have a version of "otitis" that is broad enough to encompass both "otitis media" and "otitis externa".

**Comment by cmrn-rhi** [comment]
@pnrobinson regarding my previous comment - would HPO consider reopening this request to have a general "otitis term" that could encompass "otitis externa", "otitis media", and "otitis interna"? The subclass terms are not needed for our use case at the moment, but a non-specific form of "otitis" is.
Re: at this time the SNOWMD term according to (https://www.findacode.com/snomed/43275000--otitis.html) isn't a match with "otitis externa" but rather the parent of it and "otitis media".

**Comment by cmrn-rhi** [comment]
Sorry, I now see a general "otitis" is available in MONDO. I withdraw the request to revisit this.

**Comment by pnrobinson** [comment]
No problem. Note these are not really that similar diseases.

**Comment by cmrn-rhi** [comment]
True, and while we would generally prefer a more generic "Phenotypic abnormality" version for our use case (a sign of inflammation in the ear without the specific region specified), I'll see if we can manage with the MONDO term and not put curation burden on HPO.
